### PR TITLE
Cli testing

### PR DESCRIPTION
**Pull request by elsapeng**

### CODE REVIEW

File: lib/commands/project/project.js [hunk 1]
```diff
@@ -363,3 +372,61 @@ function spawn(command, options, cwd, cb) {
...
+      var config = {
+        pathName: desiredPaths,
+        testModule: testModule,
+        assertionFormat: assertionFormat
+      };
+
+      var result = template.testGen(project.api.swagger, config);
+
+      //output the result
+      async.eachSeries(result, function (item) {
```
Comment (mohsen1) [inline lib/commands/project/project.js]: get the callback in this function arguments and pass it to `fs.outputFile`

File: lib/commands/project/project.js [hunk 2]
```diff
@@ -363,3 +372,61 @@ function spawn(command, options, cwd, cb) {
...
+        var p = item.name;
+        var filename = p.substring(1);
+        fs.outputFile(path.join(directory + '/test', filename), item.test, cb);
+      }, function(err) {
```
Comment (mohsen1) [inline lib/commands/project/project.js]: Then simply pass `cb` as the second argument to `async.eachSeries`

File: lib/commands/project/project.js [hunk 3]
```diff
@@ -363,3 +372,61 @@ function spawn(command, options, cwd, cb) {
...
+
+      // process the paths option
+      if (options.pathName){
+        var reg = new RegExp(options.pathName);
+        desiredPaths = pathList.filter(function(val) {
+          return val.match(reg);
+        });
+      }
+
+      // get the result
```
Comment (mohsen1) [inline lib/commands/project/project.js]: Comment doesn't describe what's going on here...

File: lib/commands/project/project.js (hunk 1 above)
Comment (elsapeng) [inline lib/commands/project/project.js]: changed.

File: lib/commands/project/project.js (hunk 2 above)
Comment (elsapeng) [inline lib/commands/project/project.js]: changed.

File: lib/commands/project/project.js (hunk 3 above)
Comment (elsapeng) [inline lib/commands/project/project.js]: changed

**Pull request closed without merging**